### PR TITLE
Add os_info triaged command

### PR DESCRIPTION
**Pull request by palaciosjeremias**
|Related issue|
|---|
|#8165|

## Description
This PR adds a WazuhDB command to set **os_info** table from **agents** DB to **triaged**.
It includes the UT for the new methods.
And also implements the usage of this command during the scan to avoid checking again the scanned Operating System.

## Tests

- Compilation without warnings in every supported platform
  - [X] Linux
- [X] Source installation
- [X] Package installation
- [X] Source upgrade
- [X] Package upgrade

- Memory tests for Linux
  - [X] Scan-build report
  - [X] Valgrind (memcheck and descriptor leaks check)

### CODE REVIEW

**Comment by palaciosjeremias** [comment]
WazuhDB UT:
image

**Comment by palaciosjeremias** [comment]
Valgrind on WazuhDB UT:
image

File: src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h [hunk 1]
```diff
@@ -8,23 +8,26 @@
  */
 
 
-#ifndef WDB_AGENTS_HELPERS_WRAPPERS_H
-#define WDB_AGENTS_HELPERS_WRAPPERS_H
+#ifndef WDB_AGENTS_WRAPPERS_H
```
Comment (pereyra-m) [inline src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h:11]: Now this define is the same than the file `wazuh/wazuh_db/wdb_agents_wrappers.h`. Is it a problem ?

**Review by pereyra-m — APPROVED** [review]
LGTM!

File: src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h (hunk 1 above)
Comment (DProvinciani) [reply src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h:11]: +1

File: src/unit_tests/wazuh_db/test_wdb_agents.c [hunk 2]
```diff
@@ -893,6 +921,9 @@ int main()
         /* Tests wdb_agents_clear_vuln_cves */
         cmocka_unit_test_setup_teardown(test_wdb_agents_clear_vuln_cves_statement_init_fail, test_setup, test_teardown),
         cmocka_unit_test_setup_teardown(test_wdb_agents_clear_vuln_cves_success, test_setup, test_teardown),
+        /* Tests wdb_agents_clear_vuln_cves */
```
Comment (DProvinciani) [inline src/unit_tests/wazuh_db/test_wdb_agents.c:924]:
```suggestion
        /* Tests wdb_agents_set_sys_osinfo_triaged */
```

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Please, apply changes according to the comments.

File: src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h (hunk 1 above)
Comment (palaciosjeremias) [reply src/unit_tests/wrappers/wazuh/wazuh_db/wdb_agents_helpers_wrappers.h]: It's true! This was a copy/paste error.
Fixed

File: src/unit_tests/wazuh_db/test_wdb_agents.c (hunk 2 above)
Comment (palaciosjeremias) [reply src/unit_tests/wazuh_db/test_wdb_agents.c]: Fixed

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**